docs/deploy/installation.md

```diff
@@ -50,6 +50,8 @@ curl -o iam-policy.json https://raw.githubusercontent.com/kubernetes-sigs/aws-lo
 ```
 ## Add Controller to Cluster
 
+>The controller requires a certificate for its webhook server. If you have [cert-manager installed](https://cert-manager.io/docs/installation/kubernetes/) in your cluster and set the value `enableCertManager` in Helm to true, Helm will automatically generate the certificate for you and add it as a secret. However the kustomize or yaml installation does not have the capability to generate these certificates on the fly, hence the reason why you need to install the certificate manager if you do not use Helm. In the future we may look into the controller being able to generate these certificates itself.
```

Comment:
We can add this note at the top of the guide, similar to the "security updates" note.

Reply:
The first sentence is not correct. Helm will generate secret and the corresponding certificate only if `enableCertManager` is set to false (default value).

Reply:
Yes, you are right, I got cross wired, I meant to say if the `enableCertManager` in Helm is not set to true. I can rephrase that.